huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -484,6 +484,46 @@ func ResourceFgsFunction() *schema.Resource {
 					},
 				),
 			},
+			"enable_dynamic_memory": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: `Whether the dynamic memory configuration is enabled.`,
+			},
+			"is_stateful_function": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: `Whether the function is a stateful function.`,
+			},
+			"network_controller": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"trigger_access_vpcs": {
+							Type:     schema.TypeSet,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"vpc_id": {
+										Type:        schema.TypeString,
+										Required:    true,
+										Description: `The ID of the VPC that can trigger the function.`,
+									},
+								},
+							},
+							Description: `The configuration of the VPCs that can trigger the function.`,
+						},
+						"disable_public_network": {
+							Type:        schema.TypeBool,
+							Optional:    true,
+							Default:     false,
```

Comment:
```suggestion
							Default:     false,
```
The default is false.

Reply:
sure.